src/models/view/fieldView.ts

```diff
@@ -8,6 +8,7 @@ export class FieldView implements View {
     name: string = null;
     value: string = null;
     type: FieldType = null;
+    newField: boolean = false; // Marks if the filed is new and haven't been saved
```

Comment:
small spelling item in the comments: Marks if the **field** is new and **hasn't** been saved